<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemSelectDialog</class>
 <widget class="QDialog" name="ItemSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Item</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resources/editor.qrc">
    <normaloff>:/images/level.png</normaloff>:/images/level.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="Sel_TabCon_ItemType">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Sel_Tab_Block">
        <attribute name="title">
         <string>Blocks</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="Sel_Label_FilterBlock">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Sel_Label_CategoryBlock">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Sel_Text_FilterBlock"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Sel_Label_GroupsBlock">
           <property name="text">
            <string>Group:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="Sel_Combo_FiltertypeBlock">
           <item>
            <property name="text">
             <string>Text</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ID</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_CategoryBlock">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_GroupsBlock">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QListWidget" name="Sel_List_Block"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Sel_Tab_BGO">
        <attribute name="title">
         <string>BGO</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="Sel_Label_GroupsBGO">
           <property name="text">
            <string>Group:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_GroupsBGO">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Sel_Label_CategoryBGO">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_CategoryBGO">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Sel_Label_FilterBGO">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Sel_Text_FilterBGO"/>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="Sel_Combo_FiltertypeBGO">
           <item>
            <property name="text">
             <string>Text</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ID</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QListWidget" name="Sel_List_BGO"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Sel_Tab_NPC">
        <attribute name="title">
         <string>NPC</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="Sel_Label_GroupsNPC">
           <property name="text">
            <string>Group:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_GroupsNPC">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Sel_Label_CategoryNPC">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="Sel_Combo_CategoryNPC">
           <property name="maxVisibleItems">
            <number>50</number>
           </property>
           <item>
            <property name="text">
             <string>[all]</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Sel_Label_FilterNPC">
           <property name="text">
            <string>Filter:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Sel_Text_NPC"/>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="Sel_Combo_FiltertypeNPC">
           <item>
            <property name="text">
             <string>Text</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ID</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QListWidget" name="Sel_List_NPC"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Sel_Label_ExtraData">
         <property name="text">
          <string>Extra Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="Sel_DiaButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../_resources/editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Sel_DiaButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sel_DiaButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
